bundles/nl.asml.matala.product/src/nl/asml/matala/product/validation/ProductValidator.xtend

```diff
@@ -30,6 +49,115 @@ class ProductValidator extends AbstractProductValidator {
     }
     
     
+    /**
+     * Check the duplication of variables in Inputs
+     */
+     
+     @Check
+     def checkInputDuplication(Block b){ 
```

Comment:
This change seems to be a duplicate of [Validation variable declarations #37](https://github.com/dbera/OfflineMBT/pull/37/files#diff-f565b2da544c08161f23b95efff9466437ed95e7a1c920a7cb92880ae2f637adR47) and should not be part of this pull request, right?
Applies to lines 52-90